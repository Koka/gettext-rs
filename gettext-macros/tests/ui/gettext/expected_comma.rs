use gettext_macros::*;

fn main() {
    gettext!("{}" 1);
}
