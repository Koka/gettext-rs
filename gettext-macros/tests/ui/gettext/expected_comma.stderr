error: expected `,`
 --> tests/ui/gettext/expected_comma.rs:4:19
  |
4 |     gettext!("{}" 1);
  |                   ^
